tests/stores/unit/middleware/localStorage.ts

```diff
@@ -62,12 +62,12 @@ describe('middleware - local storage', (suite) => {
 	it('should load from local storage', () => {
 		global.localStorage.setItem(LOCAL_STORAGE_TEST_ID, '[{"meta":{"path":"/counter"},"state":1}]');
 		load(LOCAL_STORAGE_TEST_ID, store);
-		assert.deepEqual((store as any)._state, { counter: 1 });
+		assert.deepEqual((store as any)._state._state, { counter: 1 });
```

Comment:
A side effect of the impl is having state stored in a nested `_state._state` object. Do you think that it should probably be something like `_adapter._state`?

Reply:
`_adapter` seems reasonable to me. I've updated it